extensions/azdata/src/localizedConstants.ts

```diff
@@ -56,7 +56,7 @@ export const userRequestedInstall = localize('azdata.userRequestedInstall', "Use
 export const userRequestedUpdate = localize('azdata.userRequestedUpdate', "User requested to update Azure Data CLI using 'Azure Data CLI: Check for Update' command");
 export const userRequestedAcceptEula = localize('azdata.acceptEula', "User requested to be prompted for accepting EULA by invoking 'Azure Data CLI: Accept EULA' command");
 export const updateCheckSkipped = localize('azdata.updateCheckSkipped', "No check for new Azure Data CLI version availability performed as Azure Data CLI was not found to be installed");
-export const eulaNotAccepted = localize('azdata.eulaNotAccepted', "Microsoft Privacy statement and Azure Data CLI license terms have not been accepted, [accept the EULA](command:azdata.acceptEula) to use the features that require it.");
+export const eulaNotAccepted = localize('azdata.eulaNotAccepted', "Microsoft Privacy statement and Azure Data CLI license terms have not been accepted. Execute the command: [Azure Data CLI: Accept EULA](command:azdata.acceptEula) to accept eula to enable the features that requires Azure Data CLI.");
```

Comment:
The error message in the dialog looks really bad since it doesn't parse the links correctly. Fix it so that the message displayed is just a normal string. I'd also suggest looking into what we could do to make it easier for the user to accept the EULA without them needing to run a command.

Reply:
Ok will replace this message in the dialog with a new one using a string that does not have these parsing issues. I will use this message to raise a toast that the user can act upon.

Reply:
done